data/notices.json

```diff
@@ -1,5 +1,21 @@
 {
   "notices": [
+    {
+      "title": "(aws-events-targets): Using an imported DLQ with an event rule target throws an error",
+      "issueNumber": 28165,
+      "overview": "All users of `aws-events-targets` who created an `events.Rule` with the `SqsQueue` target and passed an imported (eg, `fromQueueArn` or `fromQueueName`) queue to the `deadLetterQueue` prop would receive an error similar to:  ```Error: Cannot assign Dead Letter Queue in region ${Token[TOKEN.35]} to the rule XYZEventRule89558AAC in region eu-west-1. Both the queue and the rule must be in the same region.```. This only affects you if you use those constructs from versions 2.110.0 through 2.115.0.",
+      "components": [
+        {
+          "name": "aws-cdk-lib/aws-events.Rule",
+          "version": "2.110.0"
+        },
+        {
+          "name": "aws-cdk-lib/aws-events-targets.SqsQueue",
+          "version": "2.110.0"
+        }
+      ],
```

Comment:
Couple things:

1) I think the version should be set between 2.110.0 and 2.115.0. Otherwise it will only be sent to people on 2.110.0 exaclty.
2) I think we should remove the component of `aws-events.Rule` -- it only affects SqsQueue right? The additional component would only serve as noise; it wouldn't grab any other users affected by this issue